services/blockrelay/v2/proposerconfig.go

```diff
@@ -53,7 +53,7 @@ type proposerConfigJSON struct {
 
 // MarshalJSON implements json.Marshaler.
 func (p *ProposerConfig) MarshalJSON() ([]byte, error) {
-	proposer := ""
```

Comment:
Similar issues in:
https://github.com/attestantio/vouch/blob/93fa86d25a0dd57f2208657aadde2923f958eb50/services/blockrelay/v2/executionconfig.go#L185

https://github.com/attestantio/vouch/blob/93fa86d25a0dd57f2208657aadde2923f958eb50/services/blockrelay/v2/proposerconfig.go#L56

https://github.com/attestantio/vouch/blob/93fa86d25a0dd57f2208657aadde2923f958eb50/services/accountmanager/wallet/service.go#L293